scripts/scil_assign_uniform_color_to_tractograms.py

```diff
@@ -95,9 +83,8 @@ def main():
         out_filenames.append('{}{}{}'.format(base, args.out_suffix, '.trk'))
     assert_outputs_exist(parser, args, out_filenames)
 
-    sft = load_tractogram_with_reference(parser, args, filename)
-    cmap = plt.get_cmap(args.colormap)
     for i, filename in enumerate(args.in_tractograms):
+        sft = load_tractogram_with_reference(parser, args, filename)
```

Comment:
@arnaudbore should probably merge this into the scilus container ASAP.  Without moving this line in the for loop, tractometry_flow just outputs a bunch of Right UF tracks in the Color_Bundles folder with different colors.